src/Server.php

```diff
@@ -18,14 +18,26 @@ public function __construct(LoopInterface $loop)
 
     public function listen($port, $host = '127.0.0.1')
     {
-        if (strpos($host, ':') !== false) {
+        $localSocket = '';
+        if (filter_var($host, FILTER_VALIDATE_IP, FILTER_FLAG_IPV4)) {
```

Comment:
> This ext. is since PHP 5.2 enabled by default

you neverthelesss should declar it, because a lot of people - especially on production servers - use non-default builds (`` --disable-all`) which are stripped down to the very least for performance and security reasons.

> Which extensions does React use that aren't declared?

from within this code-window I would say e.g. `ext-pcre`